README.md

```diff
@@ -57,17 +70,26 @@ git checkout X.X.X
 # Building in a container
 docker run --rm -v $(pwd):/source -it centos:7
 yum install -y rpm-build make rsync gcc
```

Comment:
If using builddep below then make rsync and gcc shouldn't be needed

Reply:
and probably yum-utils? is missing

Reply:
So builddep is indeed enough.
But it seems I can run it directly from within my docker, without having to manually install yum-utils that is still present.
Have you seen cases where it was not already available?

Reply:
I've addressed this, certainly safer to explicitly install yum-utils, thanks!

Reply:
just for the record, CentOS 7 has it, but 6 & 8 not:
```
$ docker run --rm -v -it centos:6 yum-builddep
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"yum-builddep\": executable file not found in $PATH": unknown.
$ docker run --rm -v -it centos:8 yum-builddep
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"yum-builddep\": executable file not found in $PATH": unknown.
```

So good that we explicitly add it

Reply:
Cool, thanks for the confirmation